runtime/src/main/cpp/Porting.cpp

```diff
@@ -77,10 +77,21 @@ void consoleErrorUtf8(const void* utf8, uint32_t sizeBytes) {
 uint32_t consoleReadUtf8(void* utf8, uint32_t maxSizeBytes) {
 #ifdef KONAN_ZEPHYR
   return 0;
-#else 
+#else
   char* result = ::fgets(reinterpret_cast<char*>(utf8), maxSizeBytes - 1, stdin);
   if (result == nullptr) return 0;
-  return ::strlen(result);
+  uint32_t length = ::strlen(result);
```

Comment:
Looks somewhat self-repeating. Why not

const char* current = result + length - 1;
bool trimming = true;
while (current != result && trimming) {
   switch (*current) {
     case '\r':  case '\n':
         *current = 0;
         break;
     default:
      trimming = false;
    }
    current--;
 }
This way we could account for other possible linefeeds. Alternative is to do that from Kotlin side with https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-end.html

Reply:
Yes, it makes sense